src/Data/Formatter/Number.purs

```diff
@@ -0,0 +1,222 @@
+-- | This module has no support of percents and currencies.
+-- | Please, note that using simple formatter that tabulates number with
+-- | zeros and put commas between thousands should be enough for everything
+-- | because one could just compose it with `flip append "%"` or whatever
+module Data.Formatter.Number
+  ( Formatter
+  , printFormatter
+  , parseFormatString
+  , format
+  , unformat
+  , formatNumber
+  , unformatNumber

```

Comment:
Same thing as with the `formatDateTime` naming here - but again I have no good suggestion for the better alternative apart from priming.